src/layouts/LinkReport/LinksReport.tsx

```diff
@@ -0,0 +1,342 @@
+import {
+  Box,
+  BreadcrumbItem,
+  Center,
+  HStack,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+  VStack,
+} from "@chakra-ui/react"
+import { Badge, Breadcrumb, Button, Link } from "@opengovsg/design-system-react"
+import { Redirect, useParams } from "react-router-dom"
+
+import { useGetStagingUrl } from "hooks/siteDashboardHooks"
+import { useGetBrokenLinks } from "hooks/siteDashboardHooks/useGetLinkChecker"
+import { useRefreshLinkChecker } from "hooks/siteDashboardHooks/useRefreshLinkChecker"
+
+import { NoBrokenLinksImage } from "assets"
+import {
+  isBrokenRefError,
+  NonPermalinkError,
+  NonPermalinkErrorDto,
+  RepoError,
+} from "types/linkReport"
+import { useErrorToast } from "utils"
+
+import { SiteViewHeader } from "../layouts/SiteViewLayout/SiteViewHeader"
+
+const getBreadcrumb = (viewablePageInCms: string): string => {
+  /**
+   * There are four main types of pages
+   * 1. /folders/parentFolder/subfolders/childFolder/editPage/page.md -> parentFolder/childFolder/page
+   * 2. /folders/parentFolder/editPage/page.md -> parentFolder/page
+   * 3. /editPage/page.md -> Feedback Form
+   * 4. /resourceRoom/resourceRmName/resourceCategory/resourceCatName/editPage/page.md -> resourceRmName/resourceCatName/page
+   */
+  const paths = viewablePageInCms.split("/")
+  let breadcrumb = paths
+    .filter((_, index) => index % 2 === 0)
+    .slice(2)
+    .join(" / ")
+    .replace(/-/g, " ")
+  if (breadcrumb.endsWith(".md")) {
+    breadcrumb = breadcrumb.slice(0, -3)
+  }
+
+  return breadcrumb
+}
```

Comment:
Hmm does the broken link checker also support homepage/navbar/contact us? The breadcrumb wouldn't work properly for those right

Reply:
arh shucks you are right, missed these out. [8837e07](https://github.com/isomerpages/isomercms-frontend/pull/1810/commits/8837e079b04d8958509ecaf6f3945927f45ff1fb) as a todo.

navbar is not affected since its a yml, currently the scanner doesnt scan. 
bug exists for homepage + contact us, will fix upstream